test/web-platform-tests/run-to-upstream-web-platform-test.js

```diff
@@ -4,64 +4,88 @@ const http = require("http");
 const path = require("path");
```

Comment:
Sigh, so much duplicated code here between the two files. No need to fix it in this PR though.